pkg/roachprod/vm/azure/azure.go

```diff
@@ -1688,3 +1699,25 @@ func MachineFamilyVersionFromMachineType(machineType string) int {
 	}
 	return -1
 }
+
+// MachineSupportsNVMe Azure supports Nvme for E series v5 machine family.
+// OS disk and network disk support nvme. Local storage do not support nvme.
```

Comment:
Would be nice to include a link to the azure docs where you found out about which machine types support NVMe.